doc/releases/changelog-dev.md

```diff
@@ -49,9 +49,10 @@
 
 <h3>Improvements</h3>
 
-* `"default.qubit"` now skips over identity operators instead performing matrix multiplication
+* `"default.qubit"` and `"default.mixed"` now skip over identity operators instead of performing matrix multiplication
```

Comment:
Looking back, we seem to have the convention of not enclosing device names in the release notes.
```suggestion
* `default.qubit` and `default.mixed` now skip over identity operators instead of performing matrix multiplication
```